recipes/imread/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "imread" %}
+{% set version = "0.6" %}
+{% set sha256 = "97364c44b9861ebd74dbf9a7cafea604a802543b3a9b463c9de68befed72d86c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - cp %LIBRARY_LIB%\libpng.lib %LIBRARY_LIB%\png.lib  # [win]
+    - python setup.py build_ext --include-dirs=${PREFIX}/include  # [unix]
+    - python setup.py install --single-version-externally-managed --record record.txt
+    - del %LIBRARY_LIB%\png.lib  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy x.x
+    - msinttypes  # [win]
+    - jpeg
+    - libpng
+    - libwebp
+    - libtiff
```

Comment:
jpeg, libpng, libtiff have to be pinned
```
- jpeg 9*
- libtiff 4.0.*
- libpng >=1.6.28,<1.7
```

You also need to add `- zlib 1.2.*` for windows. https://github.com/luispedro/imread/blob/086e8406223db915c6e27845b4b8d6cc8529b392/setup.py#L80